ledger/src/ui/test/attestation/test_attestation.c

```diff
@@ -53,7 +54,7 @@ static cx_ecfp_private_key_t G_priv_key;
 static unsigned char G_path[PUBKEY_PATH_LENGTH];
 
 // Global onboarding flag
-const unsigned char N_onboarded_ui[1];
+NON_VOLATILE unsigned char N_onboarded_ui[1];
```

Comment:
Does using this macro here make sense at all?

Reply:
This is here just to keep the same definition as in `attestation.c`. Cold be removed and the generated code would be the same, but I think it looks better if it follows the original definition. What do you think?

Reply:
Yes, I understand the motivation, but the bottomline is those are different pieces of code and a change in one side doesn't guarantee a change in the other. In other words it's an artificial replication, a copy, and it will need to be maintained. And I can see us totally missing out on that in the future.

Is there a way we can include the actual definition as opposed to just copying it?

Reply:
Fair enough. Updated definition!